gkTally/profitLoss.py

```diff
@@ -21,68 +21,70 @@ def parse_args():
 
 # string containing key words to decide if a transaction is to be considered for
 # profit loss calculation
-profit_loss = ["bought", "sold"]
-investment = ["funding", "wire"]
+profitLossIncludeList = ["bought", "sold"]
+# exclude list - transations that needs to be excluded
+excludeList = ["Bought 100 AAPL"]
+# Investment List - transactions that needs to be considered as investment
+investmentList = ["funding", "wire"]
 
 
-class transaction_finder:
+class transactionFinder:
```

Comment:
Going forward, let us follow the python documentation rules. Every class and every method should have a comment that explains what the class or method does. This comment is actually converted into a doc string and can access it run time. Refer: https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format